src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -54,7 +57,7 @@
     private final EditPersonDescriptor editPersonDescriptor;
 
     /**
-     * @param index of the person in the filtered person list to edit
+     * @param index                of the person in the filtered person list to edit
```

Comment:
there is a large chunk of spaces(?) here, not sure if it is intended

Reply:
oh,i don't know when i space here... just delete it